.coafile

```diff
@@ -1,7 +1,8 @@
 [global]
-bears = coalaBear
+bears = coalaBear, InvalidLinkBear
 files = **.yml, **.html, **.md, **.js, **.css, **.json
 ignore = vendor/**, resources/vendors/**, _site/**
+link_ignore_regex = (twitter.com|githubusercontent.com|github.com|gitlab.com|withgoogle.com|clbin.com)
```

Comment:
why do we want to ignore all of these? github links for example might point to our wiki so they should work.

Reply:
because they return 404 for any bots.

Reply:
then please add a comment mentioning that :+1:

Reply:
Here is the removal https://github.com/coala/projects/commit/6e75373176735